### DISCUSSION TITLE
How can I configure default them in Dockerfile?

### DISCUSSION THREAD

**Discussion by YAMLcase** [node-red-contrib-themes/theme-collection#121] [General]
### Details

I'm building a custom node red docker image and need to know how to add theme settings in Dockerfile.  Thanks!

**Comment by bonanitech** [comment]
Take a look at the [Home Assistant Community Add-on: Node-RED](https://github.com/hassio-addons/addon-node-red) for reference. It's a custom docker image with the theme in it.
